[FAQ] explain 2-of-3 merging rule (#347)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-Contribution.md

Especially review by experienced C# developers are vitally important, yet even typo and grammar fixes are necessary.

There are three developers who have the ability to merge code into master branch.

> Per default, they [require at least one review](https://help.github.com/en/github/administering-a-repository/about-required-reviews-for-pull-requests) by one of the other maintainers before a pull request can be merged.

Reviewer: ```suggestion
Per default, they [require at least one approving review](https://help.github.com/en/github/administering-a-repository/about-required-reviews-for-pull-requests) by one of the other maintainers before a pull request can be merged.
```

However, since all three are administrators, they can still force the merge without the approval of others, but this will raise alarm bells.

> - [Ádám Ficsor](https://github.com/nopara73) [co-founder and former CTO of [zkSnacks Ltd](https://zksnacks.com/), co-author of the [zero link Bitcoin fungibility framework](https://github.com/nopara73/ZeroLink)] is the [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [GitHub repository](https://github.com/zksnacks/walletwasabi).

Reviewer: ```suggestion
- [Ádám Ficsor](https://github.com/nopara73) [co-founder and former CTO of [zkSNACKs Ltd](https://zksnacks.com/), co-author of the [zero link Bitcoin fungibility framework](https://github.com/nopara73/ZeroLink)] is the [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [Wasabi Wallet repository](https://github.com/zksnacks/walletwasabi).
```

> - [Lucas Ontivero](https://github.com/lontivero) [lead engineer of [zkSnacks Ltd](https://zksnacks.com/)] is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [GitHub repository](https://github.com/zksnacks/walletwasabi).

Reviewer: ```suggestion
- [Lucas Ontivero](https://github.com/lontivero) [lead engineer of [zkSNACKs Ltd](https://zksnacks.com/)] is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [Wasabi Wallet repository](https://github.com/zksnacks/walletwasabi).
```

> - [Dávid Molnár](https://github.com/molnard) is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) and administrator of the [GitHub repository](https://github.com/zksnacks/walletwasabi).

Reviewer: ```suggestion
- [Dávid Molnár](https://github.com/molnard) [CTO of [zkSNACKs Ltd](https://zksnacks.com/)] is also [admin](https://help.github.com/en/articles/repository-permission-levels-for-an-organization) of the [Wasabi Wallet repository](https://github.com/zksnacks/walletwasabi).
```
